src/bundle/Resources/views/themes/admin/content_type/relation_list_form_fields.html.twig

```diff
@@ -0,0 +1,41 @@
+{% block radio_label %}
+    {% if form.parent is defined and "ibexa_form_type_location_choice" in form.parent.vars.block_prefixes and value == "1" %}
```

Comment:
Block prefix for the radio button should be defined as form **option** for the radion button. This would circumvent the need to perform this check and be more along the lines of how Symfony Form themes work.

Something along the lines of:
```php
$builder->add('radio', CheckboxType::class, [
    'block_prefix' => 'ibexa_form_type_location_choice_radio',
]);
```

Reply:
@Steveb-p This is not a `CheckboxType` (or `RadioType`) but `ChoiceType` and there is no way of changing the `block_prefix` of underlaying inputs.

Reply:
Understood. :+1: